ui/app/AppLayouts/Chat/stores/MessageStore.qml

```diff
@@ -152,4 +152,10 @@ QtObject {
         }
         return tooltip
     }
+
+    function deleteMessage(messageId) {
+        if(!messageModule)
+            return 0
```

Comment:
Is the 0 needed? maybe we can just return nothing

Reply:
Done